includes/definitions/edgecos.yaml

```diff
@@ -10,6 +10,7 @@ over:
 discovery:
     - sysObjectID:
         - .1.3.6.1.4.1.259.6.10.94
+        - .1.3.6.1.4.1.259.8.1.11.105
```

Comment:
Can you shorten this to `.1.3.6.1.4.1.259.8.1.11` ?

https://github.com/librenms/librenms/issues/9128